public/app/features/alerting/unified/mockApi.ts

```diff
@@ -443,3 +443,5 @@ export function setupMswServer() {
 
   return server;
 }
+
+export default server;
```

Comment:
Exporting here means we can use _this_ server elsewhere, and prepend handlers to this instance (which we're then working on the assumption that this instance is the one that tests will rely on)

Reply:
I have similar concerns as Gilles about this being singleton. 
I wonder how this will work when tests are run in parallel? 🤔